plugins/imagebase/plugin.js

```diff
@@ -54,12 +54,36 @@
 			parts: {
 				link: 'a'
 			},
+			init: function() {
+				if ( this.editor.plugins.link ) {
```

Comment:
It's possible to have link plugin _without_ contextmenu plugin. Therefore this condition is wrong.